modules/rosa-deleting-cluster.adoc

```diff
@@ -32,6 +46,39 @@ $ rosa init --delete-stack
 ----
 endif::sts[]
 
+ifdef::sts[]
+.  Delete the OpenID Connect (OIDC) provider that the cluster Operators use to authenticate:
++
+[source,terminal]
+----
+$ rosa delete oidc-provider -c <cluster_id> --mode auto --yes <1>
```

Comment:
Although I wrote it this way in the Jira card, can we follow the same syntax as we did in creating the cluster by leaving off `--yes` and leaving a note saying that you can use the `-y` flag to avoid the interactive prompts? Just trying to be consistent.

Reply:
Thank you for this suggestion. This is now resolved.